modules/config-fields-redis.adoc

```diff
@@ -34,34 +32,35 @@
 |===
 | Field | Type | Description
 | **USER_EVENTS_REDIS** +
-(Required) | Object | Redis connection details for user event handling
-|{nbsp}{nbsp}{nbsp}**.host**  + 
-{nbsp}{nbsp}{nbsp}(Required)| String | The hostname at which Redis is accessible + 
-  {nbsp} + 
+(Required) | Object | Redis connection details for user event handling.
+|**.host**  + 
+(Required)| String | The hostname at which Redis is accessible. + 
 **Example:** +
 `quay-server.example.com` 
-|{nbsp}{nbsp}{nbsp}**.port**  + 
-{nbsp}{nbsp}{nbsp}(Required)| Number | The port at which Redis is accessible + 
-  {nbsp} + 
-**Example:** +
-`6379`
-|{nbsp}{nbsp}{nbsp}**.password** | String | The port at which Redis is accessible + 
-  {nbsp} + 
+|**.password** | String | The password for Redis when using encrypted mode. + 
 **Example:** +
 `strongpassword` 
+|**.port**  + 
+(Required)| Number | The port at which Redis is accessible. + 
+**Example:** +
+`6379`
+| **ssl** | Boolean | Whether to enable TLS communication between Redis and Quay. Defaults to false. 
 |===
 
 
 == Example redis configuration
 
-```
+[source,yaml]
+----
 BUILDLOGS_REDIS:
     host: quay-server.example.com
     password: strongpassword
     port: 6379
+    ssl: true
 
 USER_EVENTS_REDIS:
     host: quay-server.example.com
     password: strongpassword
     port: 6379
```

Comment:
when enable ssl, the default port number is 6380, here is an example of azure redis service connection string "quay37.redis.cache.windows.net:6380,password=k2NJbsnpRqsWBdb4fhIUbYuNhi3LDHtyuAzCaNV2Zqs=,ssl=True,abortConnect=False"